### PR TITLE
Updates to contributing guide for Augury

### LINKED ISSUE #69: Clean up contribution guide

### PR DESCRIPTION
**Pull request by tyllrb**
resolves #69

### CODE REVIEW

File: guide/contribution/contribute.md [hunk 1]
```diff
@@ -1,84 +1,62 @@
...
-## Behaving yourself
+## How Do I Actually Contribute?
 
-[Code of Conduct](https://github.com/rangle/augury/blob/dev/CODE_OF_CONDUCT.md)
-
-In the interest of making the Augury project a safe and friendly
-place for people from diverse backgrounds, we'll be adhering to a [Contributor Code of Conduct](https://github.com/rangle/augury/blob/dev/CODE_OF_CONDUCT.md). As a contributor, you'll be expected to
-uphold this code as well as report unacceptable behaviour to
-[augury@rangle.io](mailto:augury@rangle.io).
+*Remember: No contribution is too small and no issue is too trivial. If you see a typo, we'd love to get a PR. We actually hate typos...they're the worst.*
```
Comment (rajinder-yadav) [inline guide/contribution/contribute.md]: Do not use bold, per style guide.
Maybe rephrase? "We actually hate typos...they're the worst." to
"We don't like typos, find them embarrassing, if you spot any please submit a correction.

File: guide/contribution/contribute.md [hunk 2]
```diff
@@ -1,84 +1,62 @@
...
+- Try to use array operations (`map`, `reduce`, `find`, etc) instead of traditional `while` or `for` loops. 
+- Use the ES6 arrow function syntax when defining your functions
+- Try and use appropriate types for your data
 
-*At the moment, we offer limited backwards compatibility until Angular 2.0 stabilizes. Please check [README](https://github.com/rangle/augury/blob/dev/README.md#supported-version) for the version Augury currently supports.*
+More information on some of these coding styles can be found in `tslint.json` file of this repository.
 
+TypeScript linter is part of our build task and will run in either `npm run build` or `npm start`
 
 ### Pull Requests
```
Comment (rajinder-yadav) [inline guide/contribution/contribute.md]: Can you do a once over all the heading to make sure the follow writing guideline.

File: guide/contribution/contribute.md [hunk 3]
```diff
@@ -87,26 +65,31 @@ The core team will be monitoring for pull requests.
 *Before* submitting a pull request, please make sure the following is done…
 
 1. Fork the repo and create your branch from `master`.
+   
 2. If you've added code that should be tested, add tests.
```
Comment (rajinder-yadav) [inline guide/contribution/contribute.md]: Change all number to use makedown for order list to be all "1. "

File: guide/contribution/contribute.md [hunk 4]
```diff
@@ -87,26 +65,31 @@ The core team will be monitoring for pull requests.
...
-### Coding Style
+### Bugs
 
-* Use semicolons;
-* Commas last,
-* 2 spaces for indentation (no tabs)
-* Prefer `'` over `"`
-* 80 character line length
-* ...
+We are using GitHub Issues for our public bugs. We will follow up with any issues and report on when we have a fix in progress. Before filing for a new task, try to make sure your problem doesn't already exist.
```
Comment (rajinder-yadav) [inline guide/contribution/contribute.md]: doesn't - fix contraction.

File: guide/contribution/contribute.md [hunk 5]
```diff
@@ -87,26 +65,31 @@ The core team will be monitoring for pull requests.
...
 
-More information can be found in `tslint.json` file of this repository.
-TypeScript linter is part of our build task and will run in either `npm run build` or `npm start`
+If you do find a new issue, the best way to get a bug fixed is to provide a test case with either one of the example apps bundled in the repo, or by making your own Angular 2.0 application illustrating the reduced use case. You can also include code snippets, though only if they illustrate the issue in a concise way. Don't forget to include the version of Angular 2.0 app you're using!
```
Comment (rajinder-yadav) [inline guide/contribution/contribute.md]: Change Angular 2.0 to just Angular through out the doc.

Comment (rajinder-yadav) [inline guide/contribution/contribute.md]: Contraction fix for:
* Don't
* you're

File: guide/contribution/contribute.md [hunk 6]
```diff
@@ -87,26 +65,31 @@ The core team will be monitoring for pull requests.
...
+
+*At the moment, we offer limited backwards compatibility until Angular 2.0 stabilizes. Please check [README](https://github.com/rangle/augury/blob/dev/README.md#supported-version) for the version Augury currently supports.
+   
+   ​
 
 ## How to Get in Touch
 
-We are using a Slack for all of our communication. If you want to contribute or need help getting started, join us on [Slack](https://augury-slack.herokuapp.com/).
+### The chat room
```
Comment (rajinder-yadav) [inline guide/contribution/contribute.md]: Capitialize "Chat" it's a noun.

File: guide/contribution/contribute.md [hunk 7]
```diff
@@ -87,26 +65,31 @@ The core team will be monitoring for pull requests.
...
+
+[Slack](https://augury-slack.herokuapp.com/)
+
+Okay, [manuals](https://github.com/rangle/augury/blob/dev/README.md) and [textbooks](https://github.com/rangle/augury/wiki) are great and all but they make for poor conversation. Feel free to [join us on Slack](https://augury-slack.herokuapp.com/)! Whether you're looking for answers to technical questions, suggestion of a new feature, or just want to share funny pictures, there's a channel for that. You can also ask us questions in private. We can't wait to hear from you.
```
Comment (rajinder-yadav) [inline guide/contribution/contribute.md]: Correct the contractions that appear in this paragraph.

File: guide/contribution/contribute.md [hunk 8]
```diff
@@ -1,84 +1,62 @@
...
+### Developing
 
-*Remember: No contribution is too small and no issue is too trivial. If you see a typo, we'd love to get a PR. We actually hate typos...they're the worst.*
+[Developer wiki](https://github.com/rangle/augury/wiki)
 
-### Bugs
+To contribute to Augury you should have a good understanding of Angular and TypeScript. Each build of Augury is done with npm and then imported into Chrome as an extension. We have [a README just for developers](https://github.com/rangle/augury/wiki),  as well [a ARCHITECTURE guide](https://github.com/rangle/augury/wiki) with a detailed technical overview.  Jumping into a new codebase can be daunting, so the ARCHITECTURE guide is your best friend. 
 
-We are using GitHub Issues for our public bugs. We will keep track on this and will to make it clear when we have an internal fix in progress. Before filing for a new task, try to make sure your problem doesn't already exist.
+### Coding Style
```
Comment (rajinder-yadav) [inline guide/contribution/contribute.md]: style in lowercase

File: guide/contribution/contribute.md [hunk 9]
```diff
@@ -1,84 +1,62 @@
...
 
-*First things first, thank you for contributing. We know
-it takes a lot of time and effort. Everything in this document is meant
-to provide you with as much insight as possible with regards to our
-contribution standards. As you read it, it is important to remember that
-these are just __guidelines__ and not rules. Please use your best
-judgement when applying them.*
+*First things first, thank you for contributing. We know it takes a lot of time and effort. Everything in this document is meant to provide you with as much insight as possible with regards to our contribution standards. As you read it, it is important to remember that these are just __guidelines__ and not rules. Please use your best judgement when applying them.*
 
 #### Table of Contents
```
Comment (rajinder-yadav) [inline guide/contribution/contribute.md]: Make sure to correct TOC heading to follow writing guideline, as well as throughout document,

File: guide/contribution/contribute.md (hunk 3 above)
Comment (tyllrb) [inline guide/contribution/contribute.md]: Not sure what you mean here...change it so that each line starts with `1.` -- why?

**Comment by rajinder-yadav** [comment]
LGTM 🚢
